createPreviewVersions.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
```

Comment:
This file looks unnecessary

Reply:
I should be able to remove this now, will do so